.github/workflows/tests.yaml

```diff
@@ -0,0 +1,11 @@
+name: tests
+on: push
+jobs:
+  run_tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Maven Action
+        uses: s4u/setup-maven-action@v1.4.0
+        with:
```

Comment:
I am using a composite action from this repo. This is mainly since it saved some time instead of setting this all up myself. Is this something we want?